akka-http/src/main/scala/akka/http/scaladsl/server/directives/RouteDirectives.scala

```diff
@@ -49,6 +49,22 @@ trait RouteDirectives {
   def complete(m: => ToResponseMarshallable): StandardRoute =
     StandardRoute(_.complete(m))
 
+  /**
+   * Completes the request using the given arguments.
+   *
+   * @group route
+   */
+  def complete[T](status: StatusCode, v: T)(implicit m: ToEntityMarshaller[T]): StandardRoute =
```

Comment:
The main `complete` directive is by-name. Could the strict version lead to surprises?